app/services/api/v1/message_service.rb

```diff
@@ -11,7 +11,20 @@ def create
         conversation = Conversation.find(@conversation_id)
         raise InvalidConversationError unless conversation.user1_id == @current_user.id || conversation.user2_id == @current_user.id
 
-        Message.create!(content: @message_params[:content], user: @current_user, conversation_id: @conversation_id)
+        @message = Message.create!(content: @message_params[:content], user: @current_user, conversation_id: @conversation_id)
+        notify
```

Comment:
You can check if message is saved before sending the notification.

Something like:

```suggestion
        return unless @message.save?

        notify
```